omise-woocommerce.php

```diff
@@ -191,6 +194,14 @@ public function register_hooks() {
 		add_action( 'omise_async_webhook_event_handler', 'Omise_Queue_Runner::execute_webhook_event_handler', 10, 3 );
 	}
 
+	/**
+	 * @since  4.1
+	 */
+	public function register_ajax_actions() {
+		add_action('wp_ajax_nopriv_fetch_order_status', 'Omise_Ajax_Actions::fetch_order_status' );
+		add_action('wp_ajax_fetch_order_status', 'Omise_Ajax_Actions::fetch_order_status' );
+	}
```

Comment:
https://developer.wordpress.org/reference/hooks/wp_ajax_nopriv__requestaction/
According to WordPress documents, `wp_ajax_nopriv_*` is needed for those non-authenticated Ajax actions for logged-out users. We need to hook 2 actions here to be able to implement the Ajax polling to keep tracking order-status at the Thank-you page.